docs/architecture/assets.md

```diff
@@ -1,58 +1,81 @@
+1. **Parallelizable exchange:**  
+    By managing ownership and transfers directly at the account level instead of relying on global structures like ERC20 contracts, accounts can exchange assets simultaneously, boosting scalability and efficiency.
 
-## Native assets
+2. **Self-sovereign ownership:**  
+   Assets are stored in the accounts directly. This ensures that users retain complete control over their assets.
 
-Native assets are data structures that follow the Miden asset model (encoding, issuance, storing). All native assets are encoded using a single `word` (4 field elements). The asset encodes both the ID of the issuing account and the asset details. 
+3. **Censorship resistance:**  
+   Users can transact freely and anonymously with no single contract or entity controlling asset transfers. This reduces the risk of censored transactions, resulting in a more open and resilient system.
 
-Having the issuer's ID encoded in the asset makes determining the type of an asset, inside and outside Miden VM, cost-efficient. And, representing the asset in a `word` means the representation is a commitment to the asset data itself. That is particularly interesting for non-fungible assets.
+4. **Flexible fee payment:**  
+   Unlike protocols that require a specific base asset for fees, Miden allows users to pay fees in any supported asset. This flexibility simplifies the user experience.
 
-### Issuance
+## What is an asset?
+
+An asset in Miden is a unit of value that can be transferred from one [account](accounts.md) to another using [notes](notes.md).
+
+Every asset is encoded using a single `Word` (4 field elements). This `Word` includes the [ID](accounts.md#id) of the issuing account and the asset details.
+
+
+## Native asset
+
+> All data structures following the Miden asset model that can be exchanged.
 
-Only specialized accounts called faucets can issue assets. As with regular accounts, anyone can create a faucet account. Faucets can issue either fungible or non-fungible assets - but not both. 
+Native assets adhere to the Miden asset model (encoding, issuance, storage). Every native asset is encoded using a single `Word` (4 field elements). This `Word` includes both the [ID](accounts.md#id) of the issuing account and the asset details.
 
-The `faucet_id` identifies the faucet and starts with a different sequence depending on the asset type, see the [account id discussion](accounts.md#account-id). The faucet's code defines rules for how assets can be minted, who can mint them etc. Conceptually, faucet accounts on Miden are similar to ERC20 contracts on Ethereum. However, there is no ownership tracking in Miden faucets.
+### Issuance
+
+> **Info**
```

Comment:
Side note: It would be nice to render this properly as an admonition. I haven't used mkdocs before, but it looks like it has support for it but it may depend on the used theme. For example, the Material Theme for mkdocs renders them like so: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types.